README.md

```diff
@@ -74,6 +74,14 @@ pyb install -x run_unit_tests
 
 Congratulations! You have installed Hydra. <\br >
 There is a CLI mode to get some interaction with the running tests.
+
+To make use of it. Make sure to add these two paths to your .bashrc.
+
+```
+echo 'export PATH="$HOME/hydra/src/main/scripts:$PATH"' >> ~/.bashrc
+echo 'export PYTHONPATH="$HOME/hydra/src/main/python:$PATH" >> ~/.bashrc
+source ~/.bashrc
+```
```

Comment:
Hi,
I followed all the steps given in the guideline and in the end "hydra cli" wasn't working. I needed to add paths for it in order to make it work. And I was inside a venv as suggested in the  guideline.